src/openvic-simulation/pop/Pop.cpp

```diff
@@ -543,6 +544,18 @@ void Pop::report_artisanal_produce(const fixed_point_t quantity) {
 	artisanal_produce_left_to_sell = quantity;
 }
 
+void Pop::hire(pop_size_t count) {
+	if (OV_unlikely(count)) {
```

Comment:
```suggestion
	if (OV_unlikely(count == 0)) {
```
Right now this will fail if `count` is non-zero?